extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -94,7 +94,7 @@ class AmpApesterMedia extends AMP.BaseElement {
      * @private {?string}
      */
     this.mediaId_ = null;
-    /** @private {Array<Function>} */
+    /** @private {!Array<Function>} */
```

Comment:
`{!Array<function()>}` would be more appropriate

Reply:
Done.